src/sinks/aws_s3/integration_tests.rs

```diff
@@ -426,7 +425,7 @@ async fn s3_flush_on_exhaustion() {
             filename_append_uuid: true,
             filename_extension: None,
             options: S3Options::default(),
-            region: RegionOrEndpoint::with_both("minio", s3_address()),
+            region: RegionOrEndpoint::with_both("us-east-1", s3_address()),
```

Comment:
same, are why are these changing to an actual endpoint?

Reply:
New version of localstack.